cmd/object-api-datatypes.go

```diff
@@ -59,6 +59,12 @@ type BucketInfo struct {
 	Created time.Time
 }
 
+// HealInfo - represents erasure coded chunks in disks
+// TBD: Final choice of datatype
+type HealInfo struct {
+	Shards []bool
```

Comment:
- ``Corrupted`` status type represents the state where ``N/2`` or more copies of ``xl.json`` are corrupted and __not__ when one of them is, right?

- ``CanHeal`` status type represents the state where there is ``N/2+1`` or more copies of ``xl.json`` 
  are present, right?